app/views/pages/accessibility.html.erb

```diff
@@ -29,17 +29,10 @@
   </p>
 
   <h2 class="govuk-heading-m">How accessible this website is</h2>
-  <p class="govuk-body">We know some parts of this website are not fully accessible:</p>
+  <p class="govuk-body">This website is fully accessible.</p>
 
-  <ul class="govuk-list govuk-list--bullet">
-    <li>Browsers cannot automatically suggest information to help a user fill in a form. This would be particularly
-        helpful when filling in a manual address.  [1.3.5 Identify Input Purpose - Date to fix - 01.03.2023]</li>
-    <li>When an error is generated the specific error field does not have the focus. Although the tab key can used
-        it is confusing as a screen reader doesn't know where to move to. [2.4.7 Focus Visible - Date to fix - 01.03.2023]</li>
-  </ul>
-
-  <h2 class="govuk-heading-m">Non-accessible content</h2>
-  <p class="govuk-body">We link to some PDF documents that are not fully accessible to screen reader software [2018 regulations]</p>
+  <h2 class="govuk-heading-m">Disproportionate burden</h2>
+  <p class="govuk-body">Non-accessible content: In the service we use PDF documents that are not fully accessible to screen reader software [2018 regulations]</p>
```

Comment:
This is slightly different from what is in the requirements (colon instead of dash and missing full stop). Not sure how important that is